src/Test.EndToEnd.Baselining/BaseliningDetailLogger.cs

```diff
@@ -46,6 +47,12 @@ private static string RelativeSeriesPath(string seriesPath)
 
         public void Write(SarifLog newBaselineLog, SarifLog baselineLog, SarifLog currentLog, BaseliningSummary summary)
         {
+            Dictionary<string, Result> baselineResultsByGuid = new Dictionary<string, Result>();
```

Comment:
>Dictionary<string, Result> [](start = 12, length = 26)

When we rerun our style guidelines, this explicit type declaration will be converted to 'var'. I guess as a rule we no longer need to raise this sort of comment in a PR, though, right? i.e., we can just rerun that standard periodically and take the automated clean-up.

Reply:
I thought it was removing var for ambiguous cases, and not converting to var in any case. Either way, if the formatter changes it, that's fine.

I ran the formatting command earlier and it didn't seem to change anything, and I believe it was after this line was here.